tensorflow/tensorflow-model-zoo/build.gradle

```diff
@@ -13,7 +13,7 @@ dependencies {
     testImplementation("org.testng:testng:${testng_version}") {
         exclude group: "junit", module: "junit"
     }
-    testRuntimeOnly "ai.djl.tensorflow:tensorflow-native-auto:${tensorflow_version}"
+    testRuntimeOnly "ai.djl.tensorflow:tensorflow-native-auto:${tensorflow_version}-SNAPSHOT"
```

Comment:
If the public version pushlished, do we still need to keep SNAPSHOT?

Reply:
We can change it back once we released 2.3.1 to maven